libgnucash/engine/SchedXaction.hpp

```diff
@@ -0,0 +1,36 @@
+/********************************************************************\
+ * This program is free software; you can redistribute it and/or    *
+ * modify it under the terms of the GNU General Public License as   *
+ * published by the Free Software Foundation; either version 2 of   *
+ * the License, or (at your option) any later version.              *
+ *                                                                  *
+ * This program is distributed in the hope that it will be useful,  *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the    *
+ * GNU General Public License for more details.                     *
+ *                                                                  *
+ * You should have received a copy of the GNU General Public License*
+ * along with this program; if not, contact:                        *
+ *                                                                  *
+ * Free Software Foundation           Voice:  +1-617-542-5942       *
+ * 51 Franklin Street, Fifth Floor    Fax:    +1-617-542-2652       *
+ * Boston, MA  02110-1301,  USA       gnu@gnu.org                   *
+ *                                                                  *
+\********************************************************************/
+
+#ifndef XACC_SCHEDXACTION_HPP
+#define XACC_SCHEDXACTION_HPP
+
+#include "qof.h"
+#include "gnc-engine.h"
+
+#include <vector>
+#include "SX-ttinfo.hpp"
+
+/** \brief Set the schedxaction's template transaction.  tt_vec is a
```

Comment:
Since you are using doxygen's `\brief` tag here, you may as well use `\param` to explain the tt_vec parameter.